server/src/test/kotlin/org/javacs/kt/CompiledFileTest.kt

```diff
@@ -16,7 +16,7 @@ class CompiledFileTest {
         }
     }
 
-    fun compileFile(): CompiledFile = Compiler(setOf(), setOf()).use { compiler ->
+    fun compileFile(): CompiledFile = Compiler(setOf(), setOf(), outputDirectory = Files.createTempDirectory("klsBuildOutput").toFile()).use { compiler ->
```

Comment:
Since we pass the `outputDirectory` to the `Compiler`, I think it would make sense to have `CompiledFileTest` "own" the tmpdir, i.e. perhaps make it an attribute and add an `@AfterClass` or `@After` handler that automatically deletes it.